plugin/PDFDocumentPage.qml

```diff
@@ -55,4 +61,11 @@ DocumentPage {
         interval: 5000;
         onTriggered: linkArea.sourceSize = Qt.size( base.width, pdfCanvas.height );
     }
+
+    Notification {
+        id: notification;
+        category: "x-sailfish.sailfish-office.error";
+        previewBody: "Cannot open PDF file";
+        previewSummary: "Broken PDF file";
+    }    
```

Comment:
This need to be translatable:

//% "Cannot open PDF file"
previewBody:qsTrId(... ;

check other translations in this project to match the same naming.